nilearn/datasets/utils.py

```diff
@@ -22,7 +22,7 @@
 from .._utils import fill_doc
 
 _REQUESTS_TIMEOUT = (15.1, 61)
-
+_PACKAGE_DIRECTORY = os.path.dirname(os.path.abspath(__file__))
```

Comment:
Why is this added ?

Reply:
I think it comes from applying the same logic as what was done here:

https://github.com/nilearn/nilearn/blob/3e4652fba970b26b9e66c77a24ebb1655e2838de/nilearn/datasets/struct.py#L22-L32